README.md

```diff
@@ -40,41 +38,47 @@ greeting
 #=> Hi, Ruby Programmer
 ```
 
-As amazing as this method is, it's still pretty static. It hard-codes, or
+As amazing as this method is, it's still pretty _static_. It hard-codes, or
 directly specifies, the name of the person we are greeting as `"Ruby
 programmer"`. If we wanted to build a method that can greet _anyone_ by name,
 we'd need to be able to add input into the body of the method.
 
-Just like with a real-world greeting, we'd want our method to be more dynamic,
+Just like with a real-world greeting, we'd want our method to be more _dynamic_,
 more abstract, and _more re-usable_. Remember our DRY (Don't Repeat Yourself)
 principle? The method should maintain the elements that will always be the same,
 no matter who we greet, but allow us to change, or swap out, the name of the
-person we are greeting. This is "dynamic", as opposed to "hard-coded".
+person we are greeting. This is considered _dynamic_, or able to adapt to change
+or progress, as opposed to _static_, which in the case of programming means
+"hard-coded".
 
-## Define methods that accept arguments
+## Define a Methods That Accepts Arguments
 
-Now that we see the limitations of basic methods, how do we take the information
+Now that we see the limitations of static methods, how do we take the information
 we know about arguments and apply it to real-world scenarios like that of a
 doctor's office or a coffee shop?
 
-To add arguments to a method, you specify them in he line that starts with
+To add arguments to a method, you specify them in the line that starts with
```

Comment:
Careful:

To define the arguments you expect a method to take you specify **parameters** in the line....